<launch>

  <arg name="arm"/>


  <rosparam command="load" file="$(find hrl_tactile_controller)/mpc_params.yaml" />
  <rosparam command="load" file="$(find hrl_tactile_controller)/pr2_params.yaml" />

  <node name="mpc_teleop" pkg="hrl_tactile_controller" type="mpc_teleop_rviz.py" args="-r pr2 --orientation" output="screen"/>
  <node name="arm_trajectory_generator" pkg="hrl_tactile_controller" type="arm_trajectory_generator.py" args="-r pr2 -s fabric -a $(arg arm)" output="screen"/>
  <node name="robot_haptic_state" pkg="hrl_tactile_controller" type="robot_haptic_state_node.py" args="-r pr2 -s fabric -a $(arg arm)" output="screen"/>
  <node name="haptic_mpc" pkg="hrl_tactile_controller" type="haptic_mpc.py" args="-r pr2 -a $(arg arm)" output="screen"/>


</launch>
